opendbc/car/hyundai/values.py

```diff
@@ -307,6 +307,12 @@ class CAR(Platforms):
     HYUNDAI_SONATA.specs,
     flags=HyundaiFlags.MANDO_RADAR | HyundaiFlags.CHECKSUM_CRC8 | HyundaiFlags.HYBRID,
   )
+  HYUNDAI_SONATA_HEV_2024 = HyundaiCanFDPlatformConfig(
+    [HyundaiCarDocs("Hyundai Sonata Hybrid 2024", "All", car_parts=CarParts.common([CarHarness.hyundai_a]))],
+    CarSpecs(mass=1616, wheelbase=2.84, steerRatio=13.27 * 1.15, tireStiffnessFactor=0.65),
+    # TODO: confirm steerRatio increase
+    # TODO: confirm tireStiffnessFactor
```

Comment:
can you resolve these?

Reply:
Only a handfull of cars have the steerRatio factor, no obvious reason why we have that set, so removing and using cars spec as is.

I'm getting the following after a long drive, so removing stiffnessFactor and using default value since i'm not sure where were getting it from.
'stiffnessFactor': 0.9963746666908264,
'stiffnessFactorStd': 0.007022421807050705,